src/SFA.DAS.Provider.PR.Web/Models/AddEmployer/SearchByEmailViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace SFA.DAS.Provider.PR.Web.Models.AddEmployer;
+
+public class SearchByEmailViewModel : SearchByEmailSubmitViewModel
+{
+    public string CancelLink { get; }
+    public string BackLink { get; }
+    public long Ukprn { get; set; }
+
+    public SearchByEmailViewModel(string cancelLink, string backLink, int ukprn)
+    {
+        CancelLink = cancelLink;
+        BackLink = backLink;
+        Ukprn = ukprn;
+    }
```

Comment:
Instead of this type of constructors, you can use required keyword on properties or use init setters.